README.md

```diff
@@ -48,92 +48,39 @@ Please consider to donate and become a backer. 🙏 [[Become a backer](https://o
 - Pipeline steps can be named as you like
 - Run any command in the commands section
 
-```yaml
-# .woodpecker.yml
-pipeline:
-  build:
-    image: debian
-    commands:
-      - echo "This is the build step"
-  a-test-step:
-    image: debian
-    commands:
-      - echo "Testing.."
-```
+[Read More](https://woodpecker-ci.org/docs/usage/intro)
 
 ### Build steps are containers
 
 - Define any Docker image as context
 - Install the needed tools in custom Docker images, use them as context
 
-```diff
- pipeline:
-   build:
--    image: debian
-+    image: mycompany/image-with-awscli
-     commands:
-       - aws help
-```
-
-### File changes are incremental
-
-- Woodpecker clones the source code in the beginning pipeline
-- Changes to files are persisted through steps as the same volume is mounted to all steps
-
-```yaml
-# .woodpecker.yml
-pipeline:
-  build:
-    image: debian
-    commands:
-      - touch myfile
-  a-test-step:
-    image: debian
-    commands:
-      - cat myfile
-```
-
-### Plugins are straightforward
-
-- If you copy the same shell script from project to project
-- Pack it into a plugin instead
-- And make the yaml declarative
-- Plugins are Docker images with your script as an entrypoint
-
-```Dockerfile
-# Dockerfile
-FROM laszlocloud/kubectl
-COPY deploy /usr/local/deploy
-ENTRYPOINT ["/usr/local/deploy"]
-```
-
-```bash
-# deploy
-kubectl apply -f $PLUGIN_TEMPLATE
-```
-
-```yaml
-# .woodpecker.yml
-pipeline:
-  deploy-to-k8s:
-    image: laszlocloud/my-k8s-plugin
-    template: config/k8s/service.yml
-```
+[Read More](https://woodpecker-ci.org/docs/usage/pipeline-syntax#steps)
+
+### Plugins
+
+Woodpecker has official plugins https://woodpecker-ci.org/plugins,
+but you can also use own one.
```

Comment:
```suggestion
but you can also use your own.
```

Reply:
Also, should `official plugins` just be a markdown link to https://woodpecker-ci.org/plugins ?